src/common/FrameManager.ts

```diff
@@ -1204,76 +1181,60 @@ export class Frame {
+   * Focuses the first element that matches the `selector`.
    *
-   * @remarks
-   * If there's no element matching `selector`, the method throws an error.
-   *
-   * @param selector - the selector for the element to focus. If there are
-   * multiple elements, the first will be focused.
+   * @param selector - The selector to query for.
+   * @throws Throws if there's no element matching `selector`.
    */
   async focus(selector: string): Promise<void> {
-    return this._secondaryWorld.focus(selector);
+    return this.worlds[PUPPETEER_WORLD].focus(selector);
   }
 
   /**
-   * This method fetches an element with `selector`, scrolls it into view if
-   * needed, and then uses {@link Page.mouse} to hover over the center of the
-   * element.
-   *
-   * @remarks
-   * If there's no element matching `selector`, the method throws an
+   * Hovers the pointer over the center of the first element that matches the
+   * `selector`.
    *
-   * @param selector - the selector for the element to hover. If there are
-   * multiple elements, the first will be hovered.
+   * @param selector - The selector to query for.
+   * @throws Throws if there's no element matching `selector`.
    */
   async hover(selector: string): Promise<void> {
-    return this._secondaryWorld.hover(selector);
+    return this.worlds[PUPPETEER_WORLD].hover(selector);
   }
 
   /**
-   * Triggers a `change` and `input` event once all the provided options have
-   * been selected.
-   *
-   * @remarks
-   *
-   * If there's no `<select>` element matching `selector`, the
-   * method throws an error.
+   * Selects a set of value on the first `<select>` element that matches the
+   * `selector`.
    *
    * @example
    * ```ts
    * frame.select('select#colors', 'blue'); // single selection
    * frame.select('select#colors', 'red', 'green', 'blue'); // multiple selections
    * ```
    *
-   * @param selector - a selector to query the frame for
-   * @param values - an array of values to select. If the `<select>` has the
+   * @param selector - The selector to query for.
+   * @param values - The array of values to select. If the `<select>` has the
    * `multiple` attribute, all values are considered, otherwise only the first
    * one is taken into account.
    * @returns the list of values that were successfully selected.
+   * @throws Throws if there's no `<select>` matching
+   * `selector`.
    */
   select(selector: string, ...values: string[]): Promise<string[]> {
-    return this._secondaryWorld.select(selector, ...values);
+    return this.worlds[PUPPETEER_WORLD].select(selector, ...values);
   }
 
   /**
-   * This method fetches an element with `selector`, scrolls it into view if
-   * needed, and then uses {@link Page.touchscreen} to tap in the center of the
-   * element.
-   *
-   * @remarks
-   *
-   * If there's no element matching `selector`, the method throws an error.
+   * Taps the first element that matches the `selector`.
```

Comment:
should we me mention automatic scroll into view? (if it really happens)

Reply:
Let's not remark it. It's expected that this will scroll into view (in fact, all of these types of functions do). It wouldn't make sense otherwise because someone could use a function that scrolls into view, and then use a function that doesn't which would be breaking.

Reply:
But how would the users know if it scroll into the view by default or not? is there another place where it's documented?

Reply:
As I said, it's expected. Otherwise, we would have to write it for every function which means we should write it for no function.

Reply:
Imagine a new use opens Puppeteer docs and tries to see if they need to scrollIntoView before doing an action? If all the functions auto-scroll, let's mention it on the class level? For example, the `focus` method does not auto-scroll.

Reply:
Let's mention it on the class.

Reply:
that would be a breaking change though. I think it all ends up on ElementHandle and I wonder if we can just refer to the documentation there? There is a bunch of other methods that don't autoscroll: I think type and press for example.

Reply:
Yes, let's just reference it using `{@see ...}`.

Reply:
Seeing as the `ElementHandle` documentation isn't that great. I'll do this in another PR.